integration/examples/kustomize/skaffold.yaml

```diff
@@ -1,4 +1,10 @@
 apiVersion: skaffold/v3alpha1
 kind: Config
+manifests:
+  kustomize:
+    paths:
+      - .
 deploy:
-  kustomize: {}
+  kustomize:
+    paths:
+      - .
```

Comment:
I think we should be able to/want to do this with kustomize renderer -> kubectl deployer. If we can get rid of the kustomize deploy stuff here that would be great

Reply:
ohhh I see! thanks!